data/transit/railway/station.json

```diff
@@ -0,0 +1,26 @@
+{
+    "properties": {
+      "path": "transit/railway/station",
+      "skipCollection": true,
+      "exclude": {
+        "generic": ["^railway\\s?(station)?$"]
+      }
+    },
+    "items": [
+        {
+            "displayName": "LIRR",
+            "id": "lirr-94d192",
```

Comment:
I copied this template from https://github.com/osmlab/name-suggestion-index/blob/main/data/transit/route/train.json#L831, I'm not sure how you pick IDs.